proxy/prefect_flows_runner.py

```
@@ -0,0 +1,664 @@
+    logger.info(f"wrote {profiles_dir / 'profiles.yml'}")
+
+    # dbt binary — via the running Python's bin dir. Portable across:
+    #   - EKS runner image (pip installed → /usr/local/bin)
+    #   - local prefect-proxy .venv (uv installed → .venv/bin)
+    dbt_bin = os.path.join(os.path.dirname(sys.executable), "dbt")
+
+    # 4. dbt deps — installs elementary dbt package into dbt_packages/.
+    # Idempotent; safe if already installed. Fails loudly if packages.yml
+    # doesn't declare elementary (surface: user hasn't set up elementary yet).
+    logger.info("running dbt deps")
+    subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)